examples/kind/.circleci/config.yml

```diff
@@ -22,11 +22,15 @@ jobs:
       - checkout
       - run:
           command: test/e2e-kind.sh
+          no_output_timeout: 1h
```

Comment:
I noticed the issue, too, recently. The problem was that CircleCI has a default timeout of 10 minutes which is the same as the Helm timeout we use on charts CI. 

However, chart-testing doesn't set a timeout, so Helm's default of 300 sec applies which doesn't cause this issue. Thus, I don't think we should prematurely change CircleCI timeouts in the example. If a users changes timeouts they should not how to deal with them on CircleCI.

Reply:
makes sense. closing